package.json

```diff
@@ -37,6 +32,7 @@
     "react-dom": "^18.2.0",
     "react-query": "^3.39.1",
     "react-router-dom": "^6.3.0",
+    "serverless-single-page-app-plugin": "^1.0.4",
```

Comment:
Hey!) 

I am not sure it's required to install this plugin via npm.
As described in `serverless.yml`: "Existing plugin (https://www.npmjs.com/package/serverless-single-page-app-plugin) doesn't have invalidate cache feature that often used during CI/CD events."
So automatic cache invalidation is useful feature from my perspective.
I have just used preconfigured plugin from `.serverless_plugins` folder and everything works fine out of the box.
Actually the task description frustrated me a bit))

Have you tried the way I described without installing via npm?
Anyway I have marked you review as completed with highest grade.